MAUI/DataGrid/grid-events.md

```diff
@@ -103,6 +143,19 @@ private void SfDataGrid_CellRightTapped(object sender, DataGridCellRightTappedEv
 
 N> The `CellRightTapped` event is only applicable for Windows and macOS.
 
+### CellRightTapped Command and its Parameter
+When the cell is right tapped, the `CellRightTappedCommand` binds to the command which you defined in your ViewModel. Additionally, we have the option to utilize the `CellRightTappedCommandParameter` for passing data to the Command upon its execution.
```

Comment:
The command will be invoked when a right-click mouse gesture is recognized on a cell in the SfDataGrid. The instance of `DataGridCellRightTappedEventArgs` will be passed as the default value of the command's parameter. It can be customized by setting the `CellRightTappedCommandParameter` property.

Reply:
Changed